layers/+tools/sailfish-scratchbox/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- sailfish scratchbox layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2017 Victor Polevoy
```

Comment:
Copyright should be assigned to "Sylvain Benner & Contributors", as in other layers.